src/python/pants/init/options_initializer.py

```diff
@@ -108,6 +108,21 @@ def create(cls, options_bootstrapper, build_configuration, init_subsystems=True)
         .format(global_bootstrap_options.pants_version, pants_version())
       )
 
+    pants_runtime_python_version = global_bootstrap_options.pants_runtime_python_version
+    current_python_version = '.'.join(map(str, sys.version_info[0:2]))
+    if pants_runtime_python_version and pants_runtime_python_version != current_python_version:
+      raise BuildConfigurationError(
+        'Running Pants with a different Python interpreter version than requested. '
+        'You requested {}, but are running with {}.\n\n'
+        'Note that Pants cannot use the value you give for `--pants-engine-python-version` to '
```

Comment:
This is using the previous option name.

Reply:
Oof. I thought I had fixed all the instances of this. Will hotfix this as part of https://github.com/pantsbuild/pants/pull/7365. Thanks for the catch!